Source/WebCore/loader/cache/CachedFont.cpp

```diff
@@ -64,10 +68,47 @@ void CachedFont::didAddClient(CachedResourceClient& client)
         downcast<CachedFontClient>(client).fontLoaded(*this);
 }
 
+static AtomString hashForFontData(const uint8_t* data, size_t size)
+{
+    auto cryptoDigest = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
+    cryptoDigest->addBytes(data, size);
+    auto digest = cryptoDigest->computeHash();
+    return AtomString(makeString(base64Encoded(digest.data(), digest.size())));
+}
+
+static bool isAllowedFont(const AtomString& fontSha256)
```

Comment:
Not sure we need a utility function just for a single line doing a hash map look up.